src/Common/Select.php

```diff
@@ -612,12 +612,14 @@ public function groupBy(array $spec)
      *
      * @param string $cond The HAVING condition.
      *
+     * @param array ...$bind arguments to bind to placeholders
```

Comment:
Interesting question. Mixed doesn't actually make sense for it can also be array and any type. May be `string|SubselectInterface` ? We can create an issue for discussion. So others can join ?